src/Integration.Vsix.UnitTests/Analysis/AnalysisStatusNotifierTests.cs

```diff
@@ -31,36 +31,29 @@ namespace SonarLint.VisualStudio.Integration.UnitTests.Analysis
     [TestClass]
     public class AnalysisStatusNotifierTests
     {
-        private Mock<IStatusBarNotifier> statusBarMock;
```

Comment:
Changed to the new convention and updated the tests to receive the filePath argument in the CTOR.